### ISSUE TITLE
Draft R&S search filters conversation guide

### ISSUE THREAD

**Issue by davidmpickett** [department-of-veterans-affairs/va.gov-cms#19705]
## Artifacts produced
- [Conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/resources-and-support/research/2024-search-filters/conversation-guide.md)

## Dependencies

**This ticket can be worked in paralell with:**
- [ ] #19491

**This ticket blocks**
- [ ] #19706
- [ ] #20075

### User story
**AS A** Researcher
**I WANT** to document the details for conducting Resources and Support search filters research
**SO THAT** the research will achieve the stated goals and answer the research questions

### Templates

Use the appropriate template for the group you're conducting research with:
- [Editor-facing Conversation Guide Template](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/cms/research/editor-moderation-guide.md)
- [Veteran-facing Conversation Guide Template](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/research/planning/conversation-guide-template.md)

## Suggested workflow/tasks:
- [x] Brainstorm potential tasks
- [x] Create initial draft of conversation guide
- [x] Solicit review/feedback from PO, PM, and/or other team members in a public slack post
- [ ] Test prototype(s) with conversation guide and align
- [x] Post on github in project research folder for final approval before submitting to Research Review cycle

## Acceptance Criteria
- [x] Conversation guide drafted
- [ ] Feedback incorporated from PM and/or other team members
- [ ] Conversation guide aligns with prototype(s)
- [x] Conversation guide posted on github in project research folder
- [x] Final approval obtained from PO and documented at the bottom of the file in preparation for Research Review cycle

**Comment by davidmpickett** [comment]
I have started drafting conversation guide in branch: https://github.com/department-of-veterans-affairs/va.gov-team/blob/VACMS-19705-rands-convo-guide/products/resources-and-support/research/2024-search-filters/conversation-guide.md

**Comment by davidmpickett** [comment]
**End of sprint update**

- Per discussion in UX sync, I will be handing this effort off to Cindy Merrill
- I have paused work until I can start process of handing off to her

**Comment by cindymerrill** [comment]
Drafted tasks in conversation guide. Still working on questions to probe.

**Comment by cindymerrill** [comment]
Will continue drafting the conversation guide in the next sprint, @jilladams

**Comment by jilladams** [comment]
@cindymerrill could you help us out with noting what are the main tasks/ areas left to handle in the convo guide? Trying to get a sense of how far we might get in Sprint 22 (which ends 2/4),

**Comment by cindymerrill** [comment]
I just finished the convo guide and shared it in the #sitewide-ux channel for feedback from @FranECross , @davidmpickett, and @mmiddaugh . I'm especially interested in feedback on the user tasks.

@jilladams Sorry I didn't see your comment yesterday; not sure why I didn't get notified because I am subscribed to notifications. I'm done editing now and ready for feedback. I also need to test it with the prototype, which @davidmpickett is still updating.

**Comment by cindymerrill** [comment]
@jilladams Just figured out the notifications issue. They were going to my VA email, which I can't get into yet, so I updated to my agency email. Hopefully that will fix the problem...

**Comment by FranECross** [comment]
@cindymerrill I'm reviewing the conversation guide (thanks for this opportunity!), and I have to questions/suggestions. Thanks again! cc @mmiddaugh 

1. For the following question, what do you think about adding Reddit to the list of possible places? I've learned recently that many Veterans (possibly of a certain age) go there first to ask questions and get info. **Question**: When you want to learn about a program or service offered by the VA, where do you go to look for information?
Google / Facebook / MHV / eBenefits / Contact center call / email / VA.gov / _______________________



2. For the list of questions under each [If they try Show filters...], in addition to [Do they click on filter chips on search results page?], do you think it would be helpful to also check if they click the filter chips on a Details page, which will then return them to the search results using that filter? 

![Image](https://github.com/user-attachments/assets/22261c47-5cf7-412f-ab75-1e0b98bf1403)

**Comment by mmiddaugh** [comment]
Hi Cindy - thank you for selecting a wide range of tasks that will allow you to evaluate the various navigation options across a range of beneficiary types. 
Just a few thoughts/questions: 

- I noticed you have notes to observe behaviors if the participant uses filters or search - any observations or follow up questions for navigation via Featured links or topics?
- We're interested in understanding if the topic labels for Decision reviews are clear and meaningful vs Claims/Claims status - if it's possible to get insight on that by slightly adjusting one of your tasks. 
- Task 6: Sign in - would it help to add a task which would require authentication to this scenario, such as refilling a prescription or connecting it to the task of checking for debts in Task 7?
- Task 8: Discharge upgrade - I wonder if "change some of the details about how you left the military" is too vague to lead toward the intended target content. Knowing that you likely want to avoid explicitly using "discharge", would it make sense to rephrase it to include a connection to eligibility? Possibly something like, "Let's say that you want to have your service record reviewed to see if you are eligible for VA health care or other benefits" 

I'm looking forward to this research!

**Comment by cindymerrill** [comment]
Thank you, @FranECross and @mmiddaugh for your helpful feedback!

Fran: I can certainly incorporate your two suggestions, but note that the options in the warmup question are just for notetaking purposes (to circle while running the sessions), not multiple choice options that I'm going to speak aloud.

Michelle: 


> I noticed you have notes to observe behaviors if the participant uses filters or search - any observations or follow up questions for navigation via Featured links or topics?

 I didn't have anything in mind. Do you? 

> - We're interested in understanding if the topic labels for Decision reviews are clear and meaningful vs Claims/Claims status - if it's possible to get insight on that by slightly adjusting one of your tasks.

What do we want to know? Whether people can tell the difference between the two topics? Or whether they can correctly select one without being distracted by the other? The latter was my assumption, but maybe that's incorrect.

> - Task 6: Sign in - would it help to add a task which would require authentication to this scenario, such as refilling a prescription or connecting it to the task of checking for debts in Task 7?

Great idea! The task will still be about signing in, but I'll add the context for why you want to do this and got stuck. I assume that's what you mean?

> - Task 8: Discharge upgrade - I wonder if "change some of the details about how you left the military" is too vague to lead toward the intended target content. Knowing that you likely want to avoid explicitly using "discharge", would it make sense to rephrase it to include a connection to eligibility? Possibly something like, "Let's say that you want to have your service record reviewed to see if you are eligible for VA health care or other benefits"

Not sure if that's going to get people to think about discharge upgrade. Is that still content that we'd like to see if they can find? If so, maybe we could say something more explicit about their military records that might disqualify them for VA health care?

**Comment by mmiddaugh** [comment]
Re: behaviors to observe if the participant navigates via Featured links or topics?
- We'll want to know whether the Featured links are noticed and used, if they cause confusion compared to other navigational options, and if search expectations are different when a featured link is used 
- Insight on topic category labels will be useful

Re: topic labels for Decision reviews vs Claims/Claims status
- your assumptions are correct - are these viewed as distinct and meaningful categories? 

Re: adding context to Task 6 - you got it! 

Re: wording of task for Discharge upgrade - let's see if any of my OCTO content experts have any input on this when they review.

**Comment by davidmpickett** [comment]
My main focus for this round on feedback is on the choice of tasks and making sure they will serve the research goals and cover a good range of topics. Here are some thoughts:

**High-level task coverage of filter tags**
While reviewing the tasks, I [mapped them](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1731610749648/4a508431da725c30f5c99101c7402abbbef1774b) back to the benefits and topics filters to see which ones are likely to come up. (I didn't do audiences since that is covered by our recruitment approach.) 

![Image](https://github.com/user-attachments/assets/14a23d72-e072-4746-985a-4034153bfe6d)

We have really good coverage on topics. For benefits, we've definitely hit the most used benefits (health care and disability). I don't think it's feasible to cover every single benefit, but I think we might want to **add a task** that deals with the "education and training" and "careers and employment" zone. These two categories have some overlap, so it's always a good place to text from a content categorization perspective.

I was thinking it could be a more exploratory task without an explicit target article. Something like "Imagine that someone you know is getting ready to transition from active service to civilian life and has asked you to help them understand how the VA can support them. Show me how you would look for information to share with them."

If we have to cut an existing task to make room for something like this, I would say that tasks 7 and 8 are both in the lower-value zone. 

**Task 2**
I did a little brainstorming for how we might rephrase the task prompt to better elicit feedback on the distinction between the tags "Claims and claim status" and "Decision reviews and appeals"
- Consider changing the phrase "an application for VA benefits" to something more generic like "a form to the VA"
- Consider changing the phrase "an application for VA benefits" to something that specifically evokes "decision reviews and appeals" e.g. "a request to have your VA benefit application reconsidered"
- Consider changing the phrase "when you can expect to hear back from the VA" to include a purposefully misleading keyword like review or decision e.g. "when you can expect the VA to review your application" or "when you can expect the VA to make a decision"

**Comment by cindymerrill** [comment]
Thanks for your task feedback, @davidmpickett ! 

For your proposed exploratory task about transitioning, what content are you thinking they might find? Articles on educational benefits? Health care? Life insurance? I'm a little concerned that it's too open-ended, but maybe without a specific target, this could be the _first_ task so they play with the prototype a bit. When they say they've found enough information, we let them stop. I'm wondering if that will make the prototype too complex with so many articles to open up (if we want to show that), or just leave some of the articles not openable (and note which ones the participant wanted to open). We might want to start making a list of concrete search strings for the search dropdown so that participants are able to navigate this question by search, filters, and browsing.

Re "Claims and claim status" and "Decision reviews and appeals", I don't think we should include a purposely misleading keyword in the task. I think we should provide a realistic scenario and see what participants do. I'd prefer to have it suggest one of the filters or the other (such as claims status as the task is currently written), rather than have it be something generic that could be either one. But I'm not the content expert here :). We can see if they choose the "wrong" one of the two, and/or mention it as confusing during the task or at the end when we ask for feedback on the filters.

**Comment by cindymerrill** [comment]
I tried to incorporate @FranECross 's and @mmiddaugh 's feedback, while also streamlining the list of items to take note of during sessions. @davidmpickett I'd like to talk with you on Monday about user tasks and my comments above before updating them in the conversation guide, though I did add a new "Task 1.5" for the open-ended exploration task. If we want to keep it, I'll renumber all the tasks.

**Comment by cindymerrill** [comment]
Updated user tasks per conversation with @davidmpickett yesterday, and as promised made a new [spreadsheet with content and search strings desired in the prototype ](https://docs.google.com/spreadsheets/d/1-4tl2sUeBUSqJcWvjdx6_8rA1o2xOaU7pitDPad_Tk0/edit?usp=sharing)for each user task. 

Still need to add more detail pages, and consider whether we want a task for _Fraud and identity theft_ or something else that goes to a canned search.

**Comment by cindymerrill** [comment]
@davidmpickett I drafted search suggestions for each task and some detail pages, but I haven't checked what pages you have in there already. Feel free to add suggestions to the spreadsheet and/or schedule a meeting to talk together.

**Comment by cindymerrill** [comment]
Shared the research plan and conversation guide with the team + Amanda and Mikki for feedback by end-of-day Tuesday Feb 4. Feedback can go into the comments in the https://github.com/department-of-veterans-affairs/va.gov-cms/issues/19706.

**Comment by cindymerrill** [comment]
Obtained final approval from Michelle. Waiting for feedback from Fran and Mikki. Started testing prototype with user tasks but got stuck and left feedback in [Dave's ticket](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/20075#issuecomment-2628628468).

**Comment by cindymerrill** [comment]
Revised convo guide with some additional moderation prompts, per conversations with Rodney and Dave today.
